docs/sources/panels/visualizations/state-timeline.md

```diff
@@ -0,0 +1,48 @@
++++
+title = "State timeline"
+description = "Stat timeline visualization docs"
+keywords = ["grafana", "docs", "state timeline", "panel"]
+weight = 900
++++
+
+# State timeline visualization
+
+This visualization excel at showing discrete state changes over time. Each field or series is rendered as it's own unique horizontal band. State regions can either be rendered with values or without. This panel works great with string or boolean states but can also be used with time series. When used with time series the thresholds are used to turn the numerical values into discrete state regions.
+
+{{< figure src="/static/img/docs/v8/state_timeline_strings.png" max-width="1025px" caption="state timeline with string states" >}}
+
+## Shared options
+
+- [Standard options]({{< relref "../standard-options.md" >}}) (Unit, min, max, decimals, color)
+- [Thresholds]({{< relref "../thresholds.md" >}})
+- [Value mappings]({{< relref "../value-mappings.md" >}})
+
+You can also use [field overrides]({{< relref "../field-overrides.md" >}}) to specify options per field or series.
+
+## Display options
```

Comment:
Please match the formatting of other visualization option topics, particularly more recent ones like Time series, where we use headings and text rather than tables. Also, images to illustrate the options are very helpful for users.

Reply:
@oddlittlebird like this?

![Screenshot from 2021-06-01 16-52-01](https://user-images.githubusercontent.com/10999/120344293-c0124480-c2f9-11eb-8bb1-fb0089745c16.png)

does not feel great. I do no thinking images for each option is needed as they are pretty self-explanatory and adding images would just make the article a lot harder to scan and find the info you're looking for. But I can make each option a h3 heading. this table was layout was just a test, I think it made it a lot easier get an overview of the options.  But has it's drawbacks.

Reply:
pushed new headings based layout instead where each option has it's own heading